home.html

```diff
@@ -0,0 +1,183 @@
+                    </p>
+                    </div>
+                </li>
+                <li class="fSLi">
+                    <div class="fSImgContainer">
+                        <img src= "https://summit.cckorea.org/images/base/speaker_04.png" alt="tezafutati" class="fSImg">
+                    </div>
+                    <div class="fSImgInfos">
+                    <h1 class="fSLiTitle">Mamadou Shakho</h1>
+                    <h2 class="fSLiTitle2"> Minister of the environement
+                    </h2>
+                    <div class="fSLiseparator"></div>
+                    <p class="fSLiPara">He is here to ensure the safety of the environment
+                    </p>
+                    </div>
+                </li>
+                <li class="fSLi">
+                    <div class="fSImgContainer">
+                        <img src="https://summit.cckorea.org/images/base/speaker_05.png" alt="tezafutati" class="fSImg">
+                    </div>
+                    <div class="fSImgInfos">
+                    <h1 class="fSLiTitle">Macky Sall</h1>
+                    <h2 class="fSLiTitle2">The president of senegal
+                    </h2>
+                    <div class="fSLiseparator"></div>
+                    <p class="fSLiPara">He is visiting the country, after being invited.
+                    </p>
+                    </div>
+                </li>
+                <li class="fSLi">
+                    <div class="fSImgContainer">
+                        <img src="https://summit.cckorea.org/images/base/speaker_06.png" alt="tezafutati" class="fSImg">
+                    </div>
+                    <div class="fSImgInfos">
+                    <h1 class="fSLiTitle"> Koly Alpha</h1>
+                    <h2 class="fSLiTitle2"> The ex president of the country
+                    </h2>
+                    <div class="fSLiseparator"></div>
+                    <p class="fSLiPara">He is a guest of honor alongside Macky Sall.
+                    </p>
+                    </div>
+                </li>
+            </ul>
+           </section>
+           <div class="partners">
+               <div class="partners-title capitalize">partner</div>
+               <div class="separator"></div>
+               <ul class="partners-Ul">
+                   <li class="partners-Li capitalize">mozilla</li>
+                   <li class="partners-Li capitalize">Google</li>
+                   <li class="partners-Li upperCase">naver</li>
+                   <li class="partners-Li capitalize">daumkakao</li>
+                   <li class="partners-Li capitalize">airbnb</li>
+               </ul>
+           </div>
+           <div class="lisence partners">
+               <div class="lisence-title">@Tagant dates festival</div>
+               <div class="lisence-title2">2021 Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                    Provident vitae laboriosam dolor voluptate aliquam quo minima quam. </div>
+
```

Comment:
- [ ] The _footer_ and _Partner_ section is missing in your mobile design. Kindly ensure to add these sections in the mobile view.

**Your bottom section (mobile version):**
![image](https://user-images.githubusercontent.com/12745474/133658182-d55a9e90-39c4-4e4d-9872-52d8d28b1b1b.png)

**The original bottom section (mobile version):**
![image](https://user-images.githubusercontent.com/12745474/133658045-629776f5-86e3-4463-b256-735523a36cc9.png)

Reply:
- [ ] You still have to work on this